Sources/DbPackages-postgresql.php

```diff
@@ -150,28 +151,28 @@ function smf_db_create_table($table_name, $columns, $indexes = array(), $paramet
 	// If we've got this far - good news - no table exists. We can build our own!
 	if (!$db_trans)
 		$smcFunc['db_transaction']('begin');
-	$table_query = 'CREATE TABLE ' . $table_name . "\n" . '(';
+	$table_query = 'CREATE TABLE ' . $short_table_name . "\n" . '(';
 	foreach ($columns as $column)
 	{
 		// If we have an auto increment do it!
 		if (!empty($column['auto']))
 		{
 			if (!$old_table_exists)
 				$smcFunc['db_query']('', '
-					DROP SEQUENCE IF EXISTS ' . $table_name . '_seq',
+					DROP SEQUENCE IF EXISTS ' . $short_table_name . '_seq',
 					array(
 						'security_override' => true,
 					)
 				);
 
 			if (!$old_table_exists)
 				$smcFunc['db_query']('', '
-					CREATE SEQUENCE ' . $table_name . '_seq',
+					CREATE SEQUENCE ' . $short_table_name . '_seq',
```

Comment:
i still would like to see that sequence with owned get created here,
like done in this pr: https://github.com/SimpleMachines/SMF2.1/pull/6323

Reply:
Since that PR wasn't merged.  I don't think this PR should address it.  Even if it is right.

Reply:
I agree with @albertlast that an OWNED BY would be appropriate here.

In pg, sequences are standalone things, and must be associated with a column explicitly.  Not like in mysql with AUTO INCREMENT.  In mysql, if you delete the table, the auto increment goes away.  This doesn't happen in pg in SMF today, because we created all the sequences without specifying owners.

So what's the problem???   Assume you installed a pg-based SMF forum, & you need to start over.  So, you delete all the tables and attempt to reinstall.  That produces many errors today, because - among other things - you get duplicates on all the sequences.  Making the columns the owners of the sequences ensures they get deleted when the tables are deleted.  

Admittedly - not a major or high priority issue...  But the only error-free way to reinstall a pg smf forum today is to drop & recreate the pg db.  Deleting the tables isn't enough.

(Tangent alert:  And once you get past the sequence errors, you may encounter duplicate FUNCTION errors...)

The problem with that other PR is that the approach is highly unusual - it alters the pg database directly.  The equivalent would be to add a table to mysql, but not with a CREATE TABLE command, but by adding rows in the TABLE and COLUMNS tables in mysql's information_schema...  What that PR should really do is add the OWNED BYs to the install/upgrade scripts.

Reply:
Is it acceptable to say then we should make that PR a prereq for adding it into this PR?  We don't support it today and that PR adds it.  But we don't want to half ask it in this PR with that.

Reply:
you don't need to add the pr,
you should only add the logic to the code here.

Reply:
My logic here is your PR adds the owned by for the sequences.  My PR does not touch on this at all as it addresses issues with logic inconsistencies between MySQL and PostgreSQL.    If your PR merges, then I agree with your statement and my PR should be required to have the logic for the sequence owner.  But since it isn't, I don't feel my PR should address this.

If you don't agree with this, you can ping Sesq regarding this as an individual who isn't involved in this PR yet and as the Lead Dev would be the final decision maker regarding this.